doc/source/development/contributing_environment.rst

```diff
@@ -189,11 +189,8 @@ Creating a Python environment (pip)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you aren't using conda for your development environment, follow these instructions.
-You'll need to have at least the :ref:`minimum Python version <install.version>` that pandas supports. If your Python version
-is 3.8.0 (or later), you might need to update your ``setuptools`` to version 42.0.0 (or later)
-in your development environment before installing the build dependencies::
-
-      pip install --upgrade setuptools
```

Comment:
Can you include this step in the instructions block below (eg after creating and activating the virtualenv). Or otherwise we can still include it in the requirements-dev.txt?

Reply:
> Or otherwise we can still include it in the requirements-dev.txt?

the instructions become redundant if included in requirements-dev.txt? +1 on including and removing the additional instructions